apps/ui/pages/reservation-unit/[id].tsx

```diff
@@ -267,6 +284,11 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
         activeApplicationRounds,
         termsOfUse: { genericTerms: bookingTerms },
         isPostLogin: query?.isPostLogin === "true",
+        searchDuration: Number.isNaN(Number(duration))
+          ? null
+          : Number(duration),
+        searchDate: date ? `${date}` : null,
+        searchTime: time ? `${time}` : null,
```

Comment:
`date` and `time` should be strings here why are they passed to template string?

Reply:
They can be `string`, `string[]` or `undefined` as they are taken directly from the query. Changed this to `.toString()` for (possibly?) clearer code.